docs/static/breaking-changes.asciidoc

```diff
@@ -1,310 +1,38 @@
 [[breaking-changes]]
 == Breaking changes
 
-This section discusses the changes that you need to be aware of when migrating your application to Logstash 5.0 from the previous major release of Logstash (2.x).
-
-[float]
-=== Changes in Logstash Core
-
-These changes can impact any instance of Logstash and are plugin agnostic, but only if you are using the features that are impacted.
+This section discusses the changes that you need to be aware of when migrating to Logstash 6.0.0 from the previous major releases of Logstash.
 
 [float]
 ==== Application Settings
 
-[IMPORTANT]
-Logstash 5.0 introduces a new way to <<logstash-settings-file, configure application settings>> for Logstash through a
-`logstash.yml` file.
-
-This file is typically located in `${LS_HOME}/config`, or `/etc/logstash` when installed via packages. Logstash will not be
-able to start without this file, so please make sure to pass in `--path.settings` if you are starting Logstash manually
-after installing it via a package (RPM, DEB).
-
-[source,bash]
-----------------------------------
-bin/logstash --path.settings /path/to/logstash.yml
-----------------------------------
+* The setting `config.reload.interval` has been changed to use time value strings such as `5m`, `10s` etc.
+  Previously, users had to convert this to a millisecond time value themselves. 
 
 [float]
-==== URL Changes for DEB/RPM Packages
-
-The previous `packages.elastic.co` URL has been altered to `artifacts.elastic.co`.
-Ensure you update your repository files before running the upgrade process, or
-your operating system may not see the new packages.
+==== RPM/Deb package changes
 
+* For `rpm` and `deb` release artifacts, Logstash only globs config files matching `*.conf` pattern in the `conf.d` folder. 
```

Comment:
I'm not sure that "globs" is a valid verb. Also, the statement is a grammatically ambiguous. Makes it sound like the only glob pattern that Logstash will match is *.conf in the conf.d folder. Assuming my understanding is correct, how about replacing this sentence and the one that follows with:  "For `rpm` and `deb` release artifacts, config files that match the `*.conf` glob pattern must be in the `conf.d` folder, or the files will not be loaded."